src/styles/_catalog.scss

```diff
@@ -9,14 +10,32 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (min-width: 1px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
do you need this media?
it's better to remove it, and move this `grid-template-columns` to main styles to `catalog`

Reply:
done